apps/src/lib/kits/maker/util/discountLogic.js

```diff
@@ -25,3 +35,29 @@ export function isUnit6IntentionEligible(unit6Intention) {
     Unit6Intention.YES_SPRING_2021
   ].includes(unit6Intention);
 }
+
+/**
+ * @param {string} unit6Intention
+ * @returns {boolean} True if given answer to unit 6 question
+ *   makes them eligible for a discount code, and the current
+ *   date is after the opening of the eligibility window.
+ */
+export function inDiscountRedemptionWindow(unit6Intention) {
+  const today = moment();
+
+  if (Unit6Intention.YES_SPRING_2020 === unit6Intention) {
+    return true;
+  } else if (
+    Unit6Intention.YES_FALL_2020 === unit6Intention &&
+    today > eligibilityDates.FALL_2020
```

Comment:
should this be eligibilityDates.yesFall2020 or is this constant defined somewhere else?

Reply:
...and this is why I couldn't get my tests to work 🤦‍♂ . Fixed, and tests added!